lib/WasmReader/WasmLimits.h

```diff
@@ -16,7 +16,7 @@ namespace Wasm {
         static const uint32 MaxGlobals = 1000000;
         static const uint32 MaxDataSegments = 100000;
         static const uint32 MaxElementSegments = 10000000;
-        static const uint32 MaxTableSize = 10000000;
+        static const uint32 MaxTableSize = DEFAULT_CONFIG_WasmMaxTableSize;
```

Comment:
I don't think we can really change how config flags work because it has to be top-level.
I decided to keep it there for visibility even if it is now unused